src/N98/Magento/Command/Cache/ViewCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace N98\Magento\Command\Cache;
+
+use Magento\Framework\App\CacheInterface;
+use Magento\PageCache\Model\Cache\Type as FullPageCache;
+use N98\Magento\Command\AbstractMagentoCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Validator\Exception\RuntimeException;
+
+class ViewCommand extends AbstractMagentoCommand
+{
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+
+    /**
+     * @var FullPageCache
+     */
+    private $fpc;
+
+    /**
+     * @param CacheInterface $cache
+     * @param FullPageCache $fpc
+     */
+    public function inject(
+        CacheInterface $cache,
+        FullPageCache $fpc
+    ) {
+        $this->cache = $cache;
+        $this->fpc = $fpc;
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('cache:view')
+            ->addArgument(
+                'id',
+                InputArgument::REQUIRED,
+                'Cache-ID'
+            )
+            ->addOption(
+                'fpc',
+                null,
+                InputOption::VALUE_NONE,
+                'Use full page cache instead of core cache'
+            )
+            ->addOption(
+                'unserialize',
+                null,
+                InputOption::VALUE_NONE,
+                'Unserialize output'
+            )
+            ->setDescription('Prints a cache entry');
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|void
+     * @throws RuntimeException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->detectMagento($output, true);
+        if (!$this->initMagento()) {
+            return;
+        }
+
+        /** CacheInterface|FullPageCache $cacheInstance */
+        if ($input->hasOption('fpc') && $input->getOption('fpc')) {
+            $cacheInstance = $this->fpc;
+        } else {
+            $cacheInstance = $this->cache;
+        }
+
+        $cacheData = $cacheInstance->load($input->getArgument('id'));
+        if ($input->getOption('unserialize')) {
+            $cacheData = unserialize($cacheData);
+            $cacheData = print_r($cacheData, true);
```

Comment:
For most cases (structured data), a pretty print json_encode()'d string should suffice. It's less verbose in output, too and should work well with FALSE as second parameter of unserialize().